### ISSUE TITLE
git-pr-release  finish without execute anything

### ISSUE THREAD

**Issue by hasimo** [motemen/git-pr-release#27]
I installed `git-pr-release` from rubygems and run it. But it did not do anything.

## How  to reproduce

```
$ docker run -i -t ruby:2.5.4-alpine3.9 sh -c "gem install git-pr-release --version=0.7.0 && git-pr-release --help"
Fetching multipart-post-2.0.0.gem
Fetching faraday-0.15.4.gem
Fetching addressable-2.5.2.gem
Fetching sawyer-0.8.1.gem
Fetching public_suffix-3.0.3.gem
Fetching highline-2.0.1.gem
Fetching octokit-4.13.0.gem
Fetching colorize-0.8.1.gem
Fetching git-pr-release-0.7.0.gem
Fetching diff-lcs-1.3.gem
Successfully installed multipart-post-2.0.0
Successfully installed faraday-0.15.4
Successfully installed public_suffix-3.0.3
Successfully installed addressable-2.5.2
Successfully installed sawyer-0.8.1
Successfully installed octokit-4.13.0
Successfully installed highline-2.0.1
Successfully installed colorize-0.8.1
Successfully installed diff-lcs-1.3
Successfully installed git-pr-release-0.7.0
10 gems installed
<<< SHOULD SHOW SOMETHING.
```
seems `main` function is not called  from
https://github.com/motemen/git-pr-release/blob/master/bin/git-pr-release#L412

#25 could be related.

**Comment by onk** [comment]
Sorry, I will fix it today. :pray:

**Comment by onk** [comment]
Released as v0.8.0.
